### PR TITLE
Fix missing `getTestRuleConfigs` export

### PR DESCRIPTION
**Pull request by ybiquitous**
> Which issue, if any, is this issue related to?

None.

> Is there anything in the PR that needs further explanation?

No, it's self-explanatory.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Head
+
+- Fixed: missing `getTestRuleConfigs` export.
```
Comment (Mouvedia) [inline CHANGELOG.md:5]: nit

_add missing_ instead of _missing_

File: README.md
Comment (Mouvedia) [inline README.md:1]: docs

In relation to l34, L122 should be renamed to `testRuleConfigs`.

Comment (ybiquitous) [reply README.md:1]: Thanks. Fixed with a781a20

File: CHANGELOG.md (hunk 1 above)
Comment (ybiquitous) [reply CHANGELOG.md:5]: I think keeping it as-is is better since "add" is confused with the "Added:" prefix.

**Pull request merged**